src/System.Configuration.ConfigurationManager/tests/Helpers.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+using System.Configuration;
+using Xunit;
+
+namespace System.ConfigurationTests
+{
+    public static class Helpers
+    {
+        public const string IsSaveSettingsSupported = nameof(Helpers) + "." + nameof(GetIsSaveSettingsSupported);
```

Comment:
There is no point in adding two if only one is used. That said, wouldn't it be more appropriate to put another `ActiveIssue` tag on the method as you intend to follow up?

Reply:
Good catch. I didn't realized the other one was added by you in the same PR, thought rather existed also for other tests.